### DISCUSSION TITLE
Permission denied error

### DISCUSSION THREAD

**Discussion by 2000samuel** [community/community#50283] [Other Feature Feedback, Questions, & Ideas]
I'm constantly getting a "permission denied" error every time I'm running my code. 
Does anyone know how to fix this?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @2000samuel! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
